saleor/graphql/account/tests/test_account.py

```diff
@@ -1125,6 +1145,43 @@ def test_me_query_checkout_with_inactive_channel(user_api_client, checkout):
     content = get_graphql_content(response)
     data = content["data"]["me"]
     assert not data["checkout"]
+    assert not data["checkouts"]["edges"]
+
+
+def test_me_query_checkouts_with_channel(
+        user_api_client, checkout, checkout_JPY
+):
+    query = """
+        query Me($channel: String) {
+            me {
+                checkouts(first: 10, channel: $channel) {
+                    edges {
+                        node {
+                            id
+                        }
+                    }
+                    totalCount
+                }
+            }
+        }
+    """
+
+    user = user_api_client.user
+    checkout.user = checkout_JPY.user = user
+    checkout.save()
+    checkout_JPY.save()
+
+    response = user_api_client.post_graphql(
+        query, {"channel": checkout.channel.slug}
+    )
+
+    content = get_graphql_content(response)
+    data = content["data"]["me"]
+    assert data["checkouts"]["edges"][0]["node"]["id"] == graphene.Node.to_global_id(
+        "Checkout", checkout.pk
+    )
+    assert data["checkouts"]["totalCount"] == 1
```

Comment:
Would be also good to also check that the query returned the checkout from the right channel as well.